MoviesGo/MoviesGo/MainModel/View/CollectionView/GallaryCollectionViewCell.swift

```diff
@@ -205,6 +185,18 @@ final class GallaryCollectionViewCell: UICollectionViewCell {
 
     // MARK: - public functions
 
+    public func configureCell(
+        popularity: Double,
+        overview: String,
+        release: String,
+        titel: String
+    ) {
+        popularityLabel.text = String(format: "%.1f", popularity)
+        overviewLebel.text = overview
+        titleLabel.text = titel
+        releaseLabel.text = convertDataFormatter(release)
+    }
+
     public func update(mainHost: String, posterPath: String?) {
         if let posterPath = posterPath, let url = URL(string: mainHost + posterPath) {
             URLSession.shared.dataTask(with: url) { [weak self] data, _, error in
```

Comment:
```suggestion
            URLSession.shared.dataTask(with: url) { [weak self] data, _, error in
```
этот запрос выносим в сервис

Reply:
исправил